inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/node/DataNodeServiceImpl.java

```diff
@@ -142,6 +142,20 @@ public Boolean delete(Integer id, String operator) {
             return false;
         }
 
+        return delete(entity, operator);
+    }
+
+    @Override
+    public Boolean deleteByName(String name, String operator) {
+        DataNodeEntity entity = dataNodeMapper.selectByNameAndType(name, null);
```

Comment:
There may return more than one entity.
Suggested make tye type not null.

Reply:
fixed, thx